hack/templates/00-osd-serviceorchestration.configmap.yaml.tmpl

```diff
@@ -11,6 +11,7 @@ objects:
 - apiVersion: v1
   data:
     service-orchestration.json: ${SERVICE_ORCHESTRATION_RULES}
+    rh-infra-service-orchestration.json: ${RH_INFRA_SERVICE_ORCHESTRATION_RULES}
```

Comment:
Missing definition of `RH_INFRA_SERVICE_ORCHESTRATION_RULES` in list of parameters in this template.

Reply:
Good catch, thanks. Done.